menu_trans.vim.template

```diff
@@ -1248,6 +1248,32 @@ menutrans U&nfinalise TRANSLATION\ MISSING
 menutrans &Finalise TRANSLATION\ MISSING
 " }}}
 
+" ada menu
+" ada menuitems {{{1
+" From ada.vim
+menutrans Tag TRANSLATION\ MISSING
+menutrans List TRANSLATION\ MISSING
+menutrans Jump TRANSLATION\ MISSING
+menutrans Create\ File TRANSLATION\ MISSING
+menutrans Create\ Dir TRANSLATION\ MISSING
+menutrans Highlight TRANSLATION\ MISSING
+menutrans Toggle\ Space\ Errors TRANSLATION\ MISSING
+menutrans Toggle\ Lines\ Errors TRANSLATION\ MISSING
+menutrans Toggle\ Rainbow\ Color TRANSLATION\ MISSING
+menutrans Toggle\ Standard\ Types TRANSLATION\ MISSING
+" }}}
+
+" gnat menu
+" gnat menuitems {{{1
+" From gnat.vim
+menutrans GNAT TRANSLATION\ MISSING
+menutrans Build TRANSLATION\ MISSING
+menutrans Pretty\ Print TRANSLATION\ MISSING
+menutrans Tags TRANSLATION\ MISSING
```

Comment:
We already have the menu item "Tags" from another file, in line 913. So we can delete this line.